integrated_channels/sap_success_factors/exporters/utils.py

```diff
@@ -7,25 +7,11 @@
 
 from logging import getLogger
 
-from enterprise.utils import is_course_run_enrollable
 from integrated_channels.sap_success_factors.constants import SUCCESSFACTORS_OCN_LANGUAGE_CODES
 
 LOGGER = getLogger(__name__)
 
 
-def course_available_for_enrollment(course_run):
-    """
```

Comment:
moved this method to enterprise/utils with name `is_course_run_available_for_enrollment ` to reuse